lfs/localstorage.go

```diff
@@ -0,0 +1,46 @@
+package lfs
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/github/git-lfs/vendor/_nuts/github.com/rubyist/tracerx"
+)
+
+func ClearTempObjects() {
+	filepath.Walk(LocalObjectTempDir, func(path string, info os.FileInfo, err error) error {
```

Comment:
`filepath.Walk` isn't the fastest of ways to scan a directory because it does it in lexical order. Given that it's just one directory, `Readdirnames` is the fastest method, although you need `Readdir` if you also need the `FileInfo`.